### PR TITLE
Add tests for BlivetUtils class

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Test suite for `BlivetUtils` class.

### CODE REVIEW

File: tests/blivetutils_tests/blivetutilstestcase.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    @classmethod
+    def setUpClass(cls):
+
+        orig_devs = {dev for dev in os.listdir("/dev") if re.match(r"sd[a-z]+$", dev)}
+
+        test_dir = os.path.abspath(os.path.dirname(__file__))
+        ret, _out = run_command("targetcli restoreconfig %s" % os.path.join(test_dir, "targetcli_config.json"))
+        assert ret == 0
+        time.sleep(0.1)
```
Comment (vpodzime) [inline tests/blivetutils_tests/blivetutilstestcase.py]: This should probably use ``udevadm settle`` instead.

File: tests/blivetutils_tests/blivetutilstestcase.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        devs = {dev for dev in os.listdir("/dev") if re.match(r'sd[a-z]+$', dev)}
+
+        cls.vdevs = list(devs - orig_devs)
+
+        # four new devices should be added
+        assert len(cls.vdevs) == 4
+
+        # let's be 100% sure that we pick a virtual one
```
Comment (vpodzime) [inline tests/blivetutils_tests/blivetutilstestcase.py]: virtual ones

File: tests/blivetutils_tests/blivetutilstestcase.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        for d in cls.vdevs:
+            with open('/sys/block/%s/device/model' % d) as model_file:
+                assert model_file.read().strip() == 'blivetgui_test_'
+
+    @classmethod
+    def tearDownClass(cls):
+        # remove the fake SCSI devices and their backing files
+        _ret, _out = run_command("targetcli clearconfig confirm=True")
+        subprocess.call(['targetcli', 'clearconfig confirm=True'])
```
Comment (vpodzime) [inline tests/blivetutils_tests/blivetutilstestcase.py]: why twice the same command?

File: Makefile [hunk 4]
```diff
@@ -49,7 +49,18 @@ install-requires:
 
 test: check-requires
 	@echo "*** Running unittests ***"
-	PYTHONPATH=.:tests/ xvfb-run -s '-screen 0 640x480x8 -extension RANDR' python3 -m unittest discover -v -s tests/ -p '*_test.py'
+	@status=0; \
+	$(MAKE) gui-test || status=1; \
+	$(MAKE) utils-test || status=1; \
+	exit $$status
```
Comment (vpodzime) [inline Makefile]: You can use ``.ONESHELL`` for this.

File: tests/blivetutils_tests/blivetutilstestcase.py (hunk 1 above)
Comment (vojtechtrefny) [reply tests/blivetutils_tests/blivetutilstestcase.py]: Fixed

File: tests/blivetutils_tests/blivetutilstestcase.py (hunk 2 above)
Comment (vojtechtrefny) [reply tests/blivetutils_tests/blivetutilstestcase.py]: Fixed, thanks.

File: tests/blivetutils_tests/blivetutilstestcase.py (hunk 3 above)
Comment (vojtechtrefny) [reply tests/blivetutils_tests/blivetutilstestcase.py]: Copy-paste :) Fixed

File: Makefile (hunk 4 above)
Comment (vojtechtrefny) [reply Makefile]: Fixed

**Pull request merged**